src/Database/Database.php

```diff
@@ -134,8 +135,12 @@ public function connect()
 			if ($charset) {
 				$connect .= ";charset=" . $charset;
 			}
-
-			try {
+			
+                        if ($socket) {
+                                $connect .= ";$unix_socket=" . $socket;
+                        }
+			
+                        try {
```

Comment:
```suggestion

		if ($socket) {
			$connect .= ";$unix_socket=" . $socket;
		}

		try {
```

Code standard - wrong indent